makefile

```diff
@@ -101,7 +101,17 @@ e2e-test: prepare
 	minikube image load $(prevDockerImage)
 
 	@echo "=====Running e2e tests"
-	$(if $(saveOutput),gotestsum --junitfile test/test_results/e2e-tests.xml ./test/e2e/... -count=1 -timeout 70m, go test -v -count=1 -timeout 70m ./test/e2e/...) 
+	$(if $(saveOutput),gotestsum --junitfile test/test_results/e2e-tests.xml ./test/e2e/... -count=1 -timeout 70m, go test -v -count=1 -timeout 70m ./test/e2e/...)
+
+	@echo "=====Apply daemonset to configure huge pages on minikube cluster"
+	kubectl apply -f ./test/test_data/hugepages/daemonset.yaml
+
+	@echo "=====Restart minikube to apply huge pages setting"
+	minikube stop
+	minikube start --driver=docker --kubernetes-version=$(kubernetesVersion) -n=1 --memory=$(minikubeMemory) --cpus=2
+
+	@echo "=====Running e2e tests"
+	$(if $(saveOutput),gotestsum --junitfile test/test_results/hugePages-tests.xml ./test/hugePages/... -count=1 -timeout 70m, go test -v -count=1 -timeout 70m ./test/hugePages/...)
 
 	@echo "=====Delete minikube cluster"
 	minikube delete
```

Comment:
Hugepages value needs to be set back to 0.

Reply:
added step to reset the huge pages value to 0 as default.